src/controllers/reactWebviewBaseController.ts

```diff
@@ -60,33 +58,32 @@ export abstract class ReactWebviewBaseController<State, Reducers> implements vsc
 
 	protected _getHtmlTemplate() {
 		const nonce = getNonce();
-		const scriptUri = this.resourceUrl(['mssqlwebview.js']);
-		const styleUri = this.resourceUrl(['mssqlwebview.css']);
+		const baseUrl = this.resourceUrl([]).toString() + '/';
+
 		return `
 		<!DOCTYPE html>
-				<html lang="en">
+			<html lang="en">
 				<head>
-				  <meta charset="UTF-8">
-				  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-				  <title>mssqlwebview</title>
-				  <link rel="stylesheet" href="${styleUri}">
-				  <style>
+					<meta charset="UTF-8">
+					<meta name="viewport" content="width=device-width, initial-scale=1.0">
+					<title>mssqlwebview</title>
+
+					<base href="${baseUrl}">
+				<style>
 					html, body {
 						margin: 0;
 						padding: 0px;
   						width: 100%;
   						height: 100%;
 					}
-				  </style>
+				</style>
 				</head>
 				<body>
-				  <div id="root"></div>
-				  <script>
-				 	const assetPathVscodeUri = "${this.resourceUrl(['']).toString()}";
-				  </script>
-				  <script nonce="${nonce}" src="${scriptUri}"></script>
+					<link rel="stylesheet" href="${this._sourceFile}.css">
+					<div id="root"></div>
+				  	<script type="module" nonce="${nonce}" src="${this._sourceFile}.js"></script>

```

Comment:
Since we are using 'esm' format, we have to let chrome know that the script is of type module. Without this, chrome won't support esm syntax in the script like 'import'